partitura/io/importmusicxml.py

```diff
@@ -187,6 +190,11 @@ def load_musicxml(xml, ensure_list=False, validate=False, force_note_ids=None):
         A list of either Part or PartGroup objects
 
     """
+    
+    if os.path.basename(xml).endswith(".mxl"):
```

Comment:
Perhaps it would be better to use `os.path.splitext` instead of `endswith`? (to make everything as "platform" agnostic as possible. 
Another possibility would be to use a `try` and `except` (since we cannot guarantee that the files would have the appropriate extension)

Reply:
good point!